raco/backends/myria/myria.py

```diff
@@ -1883,23 +1888,33 @@ def fire(self, op):
         self.collect_and_replace(op, {})
         return op
 
+    def __str__(self):
+        return ("Store[relation](IDBScan(IDBController)) ->"
+                "IDBController[relation]")
+
+
+def replace_child_with(op, child, replacement):
```

Comment:
Why don't we `assert` in an `else` clause rather than doing nothing for unexpected operator types.